pkgs/development/libraries/ngtcp2/default.nix

```diff
@@ -27,6 +27,11 @@ stdenv.mkDerivation rec {
     "-DENABLE_STATIC_LIB=OFF"
   ];
 
+  preConfigure = ''
+    substituteInPlace crypto/includes/CMakeLists.txt \
+      --replace 'ngtcp2/ngtcp2_crypto_openssl.h' 'ngtcp2/ngtcp2_crypto_quictls.h'
```

Comment:
Without the condition, it would still be nice to have an explanation why we need/use quictls instead of openssl here.